00-average/Average.java

```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+public class Average{
+  public static void main(String[] args) {
+    //ask for 3 test grades
+    Scanner input = new Scanner(System.in);
+    int sum = 0;
+    System.out.print("Enter the first score: ");
+    sum += input.nextInt();
+    System.out.print("Enter the second score: ");
+    sum += input.nextInt();
+    System.out.print("Enter the third score: ");
+    sum += input.nextInt();
+    double average = sum/3;
```

Comment:
does the average show accurately, like if 99 and 100 were to be entered, would the average be 99.5? Or simply 99?